vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/component/ExperimentInfoComponent.java

```diff
@@ -0,0 +1,168 @@
+package life.qbic.datamanager.views.projects.project.experiments.experiment.component;
+
+import com.vaadin.flow.component.ComponentEventListener;
+import com.vaadin.flow.component.contextmenu.MenuItem;
+import com.vaadin.flow.component.contextmenu.SubMenu;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.ListItem;
+import com.vaadin.flow.component.html.Span;
+import com.vaadin.flow.component.html.UnorderedList;
+import com.vaadin.flow.component.menubar.MenuBar;
+import com.vaadin.flow.component.menubar.MenuBarVariant;
+import java.io.Serial;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Objects;
+import life.qbic.datamanager.views.general.Card;
+import life.qbic.projectmanagement.domain.project.experiment.vocabulary.Analyte;
+import life.qbic.projectmanagement.domain.project.experiment.vocabulary.Species;
+import life.qbic.projectmanagement.domain.project.experiment.vocabulary.Specimen;
+
+/**
+ * <b>Experiment Info Component</b>
+ *
+ * <p>A component that nicely renders experiment-related general information</p>
+ *
+ * @since 1.0.0
+ */
+public class ExperimentInfoComponent extends Card {
+
+  private boolean controlsEnabled = true;
+  @Serial
+  private static final long serialVersionUID = -4790635833822470484L;
+  public static final String TITLE = "title";
```

Comment:
Why do we need a static final title here? Where is it used?

Reply:
Sonar Cloud was complaining about duplicated mention of title in three places. The solution is a constant :)